client/src/views/mapping/components/inputColumn/addJoinButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "@blueprintjs/core";
+
+import * as React from "react";
+
+import { Mutation } from "react-apollo";
+
+const createJoinAndUpdateInputColumn = require("../../../../graphql/mutations/createJoinAndUpdateInputColumn.graphql");
```

Comment:
If you merge from master, you can also use absolute imports (here, `src/graphql/......` should work)